src/app/chip_data_model.gni

```diff
@@ -149,14 +150,19 @@ template("chip_data_model") {
 
     if (defined(invoker.zap_pregenerated_dir)) {
       sources += [
-        "${invoker.zap_pregenerated_dir}/CHIPClusters.cpp",
-        "${invoker.zap_pregenerated_dir}/CHIPClusters.h",
-        "${invoker.zap_pregenerated_dir}/CHIPClustersInvoke.cpp",
-        "${invoker.zap_pregenerated_dir}/CHIPClustersWrite.cpp",
         "${invoker.zap_pregenerated_dir}/attribute-size.cpp",
         "${invoker.zap_pregenerated_dir}/callback-stub.cpp",
       ]
 
+      if (chip_python_controller) {
```

Comment:
This seems odd.  Why do other things that use controller clusters, say, not need this?

Reply:
By default `chip_python_controller` is true (https://github.com/project-chip/connectedhomeip/pull/11116/files/c1c0be1ccc506bcdbdf148a0ebef853134ea505c#diff-b3871ceca7b7fb48c3fd91178d2c0893f75735547c31eb1f7cac92f0f968f9cbR19). Only for the Python end-device it is set to false.

Possibly I should rename the variable to `chip_controller` to make it clear, that it does not only apply to the Python controller.

Reply:
Yes, renaming that way would be a lot clearer.